azure-pipelines.bundlesize.yml

```diff
@@ -47,13 +47,13 @@ jobs:
           PathtoPublish: 'packages/office-ui-fabric-react/dist/office-ui-fabric-react.js'
           ArtifactName: oufrdrop
 
-  # - job: lightrail
-  #   pool: server
-  #   dependsOn: build
-  #   steps:
-  #     - task: odefun.odsp-lightrail-tasks-partner.odsp-lightrail-tasks-SizeAuditorWorker.SizeAuditorWorker@0
-  #       displayName: 'Size Auditor Check on LightRail'
-  #       inputs:
-  #         connectedServiceName: lowimpact
-  #         sourceVersionMessage: '$(Build.SourceVersionMessage)'
-  #         sourceRepositoryUrl: 'https://github.com/OfficeDev/office-ui-fabric-react'
+  - job: lightrail
+    pool: server
+    dependsOn: build
+    steps:
+      - task: odefun.odsp-lightrail-tasks-partner.odsp-lightrail-tasks-SizeAuditorWorker.SizeAuditorWorker@0
+        displayName: 'Size Auditor Check on LightRail'
+        inputs:
+          connectedServiceName: lowimpact
+          sourceVersionMessage: '$(Build.SourceVersionMessage)'
```

Comment:
@JasonGore, from the [link](https://developercommunity.visualstudio.com/content/problem/857818/cannot-read-buildsourceversionmessage-when-check-i.html) you mentioned in PR, looks like adding double quotes could work

So, `sourceVersionMessage: "$(Build.SourceVersionMessage)"`

We should see the result when pushing the commit, so low-risk. I don't have a dev environment with me.

Reply:
Yes, I tried without quotes and with double quotes in #11504 and reported my findings in the Azure issue. None of that has worked so far.